cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/StepExecutionData.java

```diff
@@ -275,5 +286,30 @@ private String computeWorkerGroup(WorkerGroupStatement workerGroup,
         return workerGroupValue.toString();
     }
 
+    private void putStepNavigationOptions(RunEnvironment runEnv, Map<String, NavigationOptions> stepNavigationOptions) {
+        if (stepNavigationOptions != null) {
+            for (NavigationOptions options: stepNavigationOptions.values()) {
```

Comment:
why for loop?

Reply:
Here I need to fetch first value.
Instead of loop, I can use something like stepNavigationOptions.values().iterator().next().